### ISSUE TITLE
508: Account Information Page not accessible by screen readers

### ISSUE THREAD

**Issue by MitchellTCG** [fecgov/fecfile-web-app#1934]
**Business Reason**

As a user with accessibility issues, I would like to be able to use a screen reader to understand the information on the Account Information page

**Task:** Make the WHOLE PAGE screen-readable on the page Account Information page

*   “An official website of the United State…”; the hyperlink; and all hidden information
*   **Include the top navigation** (currently missing from screen read) 
      *   Dashboard
    *   Reports
    *   Contacts
    *   Tools
    *   Profile
*   Committee Header sections 
      *   Committee name
    *   Committee Status & Monthly or Quarterly
    *   Type of committee
    *   Committee ID
*   Committee (Should be updated from Contributor)
*   Address
*   Treasurer
*   Custodian
*   “You can only view your committee’s information on this page”; the two buttons
*   Footer

### Design ###
h2. Wireframe:

!Screenshot 2024-11-12 at 1.37.30 PM.png|width=1941,height=1048,alt="Screenshot 2024-11-12 at 1.37.30 PM.png"!

 See full ticket and images here: [FECFILE-392](https://fecgov.atlassian.net/browse/FECFILE-392)

Pull Request: https://github.com/fecgov/fecfile-web-app/pull/2403

**Comment by mary-tcg** [comment]
@MitchellTCG image updated
